src/sql/media/icons/common-icons.css

```diff
@@ -233,6 +233,15 @@
 	background: url("unpin.svg") center center no-repeat;
 }
 
+.hc-black .codicon.bookreport,
```

Comment:
Just call it book. Although I'm not sure this icon is really the best option for an action that generates a report - but I'll leave that choice up to you.

Reply:
"Book" is already taken by
https://github.com/microsoft/azuredatastudio/blob/6751dffbc39adff702feb2768fb2df7de26d4e45/src/sql/media/icons/common-icons.css#L558

Reply:
Hrm, really wish we didn't have this mix of masked and non-masked icons...yeah ok this is fine for now.